apps/testrunner/queue.py

```diff
@@ -0,0 +1,105 @@
+import json
+import time
+
+import requests
+import sys
+
+from .logger import logger
+from .runner import SimpleTest
+
+
+class TaskRepo(object):
+    # todo: add HTTP requests to fetch/save data
+    URL = 'http://localhost:8000/task'
+
+    def acquire(self):
+        url = '{}/acquire'.format(self.URL)
+        response = requests.get(url)
+        if response.ok:
+            task_data = response.json()
+            return Task(self, task_data['id'], task_data)
+
+    def release(self, task):
+        url = '{}/{}/lock'.format(self.URL, task.id)
```

Comment:
Just an idea: If we would like to move those URI's to an config it would probably be better to use named placeholders:

``` '{base_url}/{task_id}/lock'.format(base_url = self.URL, task_id = task.id)```